queries/heex/injections.scm

```diff
@@ -1,11 +1,15 @@
-; HEEx directives can span multiple interpolated lines of Elixir
-(directive [
-  (expression_value) 
-  (partial_expression_value)
-] @elixir @combined)  
+; directives are standalone tags like '<%= @x %>'
+;
+; partial_expression_values are elixir code that is part of an expression that
+; spans multiple directive nodes, so they must be combined. For example:
+;     <%= if true do %>
+;       <p>hello, tree-sitter!</p>
+;     <% end %>
+((directive (partial_expression_value) @elixir @combined))
 
-; HEEx Elixir expressions are always within a tag or component
-(expression (expression_value) @elixir)
+; Regular expression_values do not need to be combined
+((directive (expression_value) @elixir))
 
-; HEEx comments
-(comment) @comment
```

Comment:
why did you remove the comment injection?

Reply:
@clason is it needed in the injections? it's already in the highlights? i'm really a newbie with regards to treesitter syntax, i'll have to read up on that but it seems it's redundant here.

Reply:
Yes, it's wanted, to be able to apply `TODO` etc. highlighting in comments.

(If in doubt, don't touch it ;))